packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/rest-length/expected.js

```diff
@@ -0,0 +1,14 @@
+var t = function (f) {
+  arguments.length <= 1 ? undefined : arguments[1];
+  arguments.length <= arguments.length - 1 - 1 + 1 ? undefined : arguments[arguments.length - 1 - 1 + 1];
```

Comment:
Could his be optimized out or is the assumption that uglify will reduce it back to the simplified `args[args.length - 1]` form?